code/lib/store/template/stories/rendering.stories.ts

```diff
@@ -9,14 +9,14 @@ export default {
     label: 'Click me',
   },
 };
-
+// this is the story should be remove it will always fail, because forceReRender Api is removed in V7
 export const ForceReRender = {
   play: async ({ canvasElement }: PlayFunctionContext<any>) => {
     const channel = globalThis.__STORYBOOK_ADDONS_CHANNEL__;
     const button = await within(canvasElement).findByRole('button');
     await button.focus();
     await expect(button).toHaveFocus();
-
+    //  forceRender is not called in V7
```

Comment:
Actually this test was a 🤦 . It should be emitting `FORCE_REMOUNT`. But the bigger issue is making sure it actual fails (chromatic & the test runner) when the assertion fails like this.

Reply:
I have tested it is not trigging RORCE_RMOUNT,  renderToCanvas is not called at all